ert_shared/ensemble_evaluator/queue_adaptor.py

```diff
@@ -0,0 +1,126 @@
+import asyncio
+import copy
+import json
+import threading
+import time
+
+import websockets
+from ert_shared.ensemble_evaluator.entity import RealizationDecoder, create_realization
+from ert_shared.ensemble_evaluator.ws_util import wait_for_ws
+from res.job_queue import JobQueueManager
+from res.job_queue.job_status_type_enum import JobStatusType
+
+
+def status_equal(statuses):
+    return statuses[0] == statuses[1]
+
+
+class JobQueueManagerAdaptor(JobQueueManager):
+
+    ws_url = "ws://localhost:8765"
```

Comment:
This will eventually go away when we have a more firm concept of a legacy EE. The EE will then instantiate the adaptor itself, and just pass along its own WS.

There is no legacy EE just yet, so this is a temporary way to set it.